### PR TITLE
Switch to quay.io

### PR DESCRIPTION
**Pull request by danielhoherd**
related: astronomer/issues#1355

Switch from docker hub to quay.io, which is where Astronomer will be publishing ap docker images in the long term.

These are documentation changes only.

### CODE REVIEW

**Comment by potiuk** [comment]
I guess this should be rather changed to the "airflow" ones which we already switched the helm chart to.

File: chart/README.md [hunk 1]
```diff
@@ -142,16 +142,16 @@ The following tables lists the configurable parameters of the Airflow chart and
 | `images.flower.repository`                            | Docker repository to pull image from. Update this to deploy a custom image                                   | `~`                                               |
 | `images.flower.tag`                                   | Docker image tag to pull image from. Update this to deploy a new custom image tag                            | `~`                                               |
 | `images.flower.pullPolicy`                            | PullPolicy for flower image                                                                                  | `IfNotPresent`                                    |
-| `images.statsd.repository`                            | Docker repository to pull image from. Update this to deploy a custom image                                   | `astronomerinc/ap-statsd-exporter`                |
+| `images.statsd.repository`                            | Docker repository to pull image from. Update this to deploy a custom image                                   | `quay.io/astronomer/ap-statsd-exporter`           |
```
Comment (potiuk) [inline chart/README.md:145]:
```suggestion
| `images.statsd.repository`                            | Docker repository to pull image from. Update this to deploy a custom image                                   | `apache/airflow:airflow-statsd-exporter-2020.09.05-v0.17.0`       |
```

File: chart/README.md [hunk 2]
```diff
@@ -142,16 +142,16 @@ The following tables lists the configurable parameters of the Airflow chart and
...
 | `images.redis.repository`                             | Docker repository to pull image from. Update this to deploy a custom image                                   | `redis`                                           |
 | `images.redis.tag`                                    | Docker image tag to pull image from. Update this to deploy a new custom image tag                            | `6-buster`                                        |
 | `images.redis.pullPolicy`                             | PullPolicy for redis image                                                                                   | `IfNotPresent`                                    |
-| `images.pgbouncer.repository`                         | Docker repository to pull image from. Update this to deploy a custom image                                   | `astronomerinc/ap-pgbouncer`                      |
+| `images.pgbouncer.repository`                         | Docker repository to pull image from. Update this to deploy a custom image                                   | `quay.io/astronomer/ap-pgbouncer`                 |
```
Comment (potiuk) [inline chart/README.md:151]:
```suggestion
| `images.pgbouncer.repository`                         | Docker repository to pull image from. Update this to deploy a custom image                                   | `apache/airflow:airflow-pgbouncer-2020.09.05-1.14.0`                 |
```

File: chart/README.md [hunk 3]
```diff
@@ -142,16 +142,16 @@ The following tables lists the configurable parameters of the Airflow chart and
...
 | `images.pgbouncer.tag`                                | Docker image tag to pull image from. Update this to deploy a new custom image tag                            | `~`                                               |
 | `images.pgbouncer.pullPolicy`                         | PullPolicy for pgbouncer image                                                                               | `IfNotPresent`                                    |
-| `images.pgbouncerExporter.repository`                 | Docker repository to pull image from. Update this to deploy a custom image                                   | `astronomerinc/ap-pgbouncer-exporter`             |
+| `images.pgbouncerExporter.repository`                 | Docker repository to pull image from. Update this to deploy a custom image                                   | `quay.io/astronomer/ap-pgbouncer-exporter`        |
```
Comment (potiuk) [inline chart/README.md:154]:
```suggestion
| `images.pgbouncerExporter.repository`                 | Docker repository to pull image from. Update this to deploy a custom image                                   | `apache/airflow:airflow-pgbouncer-exporter-2020.09.25-0.5.0`        |
```

**Review by potiuk — CHANGES_REQUESTED** [review]

**Comment by danielhoherd** [comment]
I meant to PR this to the original repo, not the fork. My bad!

**Comment by potiuk** [comment]
Ah! Been there, done that :)

**Pull request closed without merging**